app/i18n/tr/gen.php

```diff
@@ -209,56 +209,56 @@
 		'years' => 'yıl',
 	),
 	'share' => array(
-		'Known' => 'Bilinen siteler',
-		'archiveIS' => 'archive.is',	// IGNORE
-		'archiveORG' => 'archive.org',	// IGNORE
-		'archivePH' => 'archive.ph',	// IGNORE
-		'bluesky' => 'Bluesky',	// TODO
-		'buffer' => 'Buffer',	// IGNORE
-		'clipboard' => 'Kopyala',
-		'diaspora' => 'Diaspora*',	// IGNORE
-		'email' => 'Email',	// IGNORE
-		'email-webmail-firefox-fix' => 'Email (webmail - Firefox için düzeltme)',
-		'facebook' => 'Facebook',	// IGNORE
-		'gnusocial' => 'GNU social',	// IGNORE
-		'jdh' => 'Journal du hacker',	// IGNORE
-		'lemmy' => 'Lemmy',	// IGNORE
-		'linkding' => 'Linkding',	// IGNORE
-		'linkedin' => 'LinkedIn',	// IGNORE
-		'mastodon' => 'Mastodon',	// IGNORE
-		'movim' => 'Movim',	// IGNORE
-		'omnivore' => 'Omnivore',	// IGNORE
-		'pinboard' => 'Pinboard',	// IGNORE
-		'pinterest' => 'Pinterest',	// IGNORE
-		'pocket' => 'Pocket',	// IGNORE
-		'print' => 'Print',	// IGNORE
-		'raindrop' => 'Raindrop.io',	// IGNORE
-		'reddit' => 'Reddit',	// IGNORE
-		'shaarli' => 'Shaarli',	// IGNORE
-		'telegram' => 'Telegram',	// IGNORE
-		'twitter' => 'Twitter',	// IGNORE
-		'wallabag' => 'wallabag v1',	// IGNORE
-		'wallabagv2' => 'wallabag v2',	// IGNORE
+		'Known' => 'Bilinen tabanlı siteler',
+		'archiveIS' => 'archive.is',
+		'archiveORG' => 'archive.org',
+		'archivePH' => 'archive.ph',
+		'bluesky' => 'Bluesky',
+		'buffer' => 'Buffer',
+		'clipboard' => 'Pano',
+		'diaspora' => 'Diaspora*',
+		'email' => 'E-posta',
+		'email-webmail-firefox-fix' => 'E-posta (webmail - Firefox için düzeltme)',
+		'facebook' => 'Facebook',
+		'gnusocial' => 'GNU social',
+		'jdh' => 'Journal du hacker',
+		'lemmy' => 'Lemmy',
+		'linkding' => 'Linkding',
+		'linkedin' => 'LinkedIn',
+		'mastodon' => 'Mastodon',
+		'movim' => 'Movim',
+		'omnivore' => 'Omnivore',
+		'pinboard' => 'Pinboard',
+		'pinterest' => 'Pinterest',
+		'pocket' => 'Pocket',
+		'print' => 'Yazdır',
+		'raindrop' => 'Raindrop.io',
+		'reddit' => 'Reddit',
+		'shaarli' => 'Shaarli',
+		'telegram' => 'Telegram'
```

Comment:
But also all of the ignores of course.
```suggestion
		'telegram' => 'Telegram',
```